### ISSUE TITLE
Remove deprecated SafeNetstringReceiver

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4546]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4546|
|Type|task|
|Created|2010-07-05 23:44:07Z|
|Branch|https://github.com/twisted/twisted/tree/remove-SafeNetstringReceiver-4546|

Looks like SafeNetstringReceiver in [twisted/protocols/basic.py](twisted/protocols/basic.py) was added as deprecated (?) in r591, let's remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__4546 4546
type__task task
reporter__thijs thijs
priority__lowest lowest
milestone__ 
branch__branches_remove_SafeNetstringReceiver_4546 branches/remove-SafeNetstringReceiver-4546
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1278373447000000 1278373447000000
changetime__1288688775000000 1288688775000000
version__None None
owner__ 
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli commented|
|-|-|

(In [30166]) Removed twisted.protocols.basic.SafeNetstringReceiver.
Refs [#4546](https://github.com/twisted/twisted/issues/4546)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli removed owner|
|-|-|

Up for review.  Strangely, trac didn't pick up the branch creation.  The branch is here:  http://twistedmatrix.com/trac/browser/branches/remove-SafeNetstringReceiver-4546

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli commented|
|-|-|

[build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-SafeNetstringReceiver-4546)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli commented|
|-|-|

[ second build where that one failing test passed](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-SafeNetstringReceiver-4546&num_builds=2)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_6"></a>|@cyli commented|
|-|-|

(In [30179]) Changed newsfile to conform with accepted format.

Refs [#4546](https://github.com/twisted/twisted/issues/4546)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess set owner to @cyli|
|-|-|

Thanks for working on this, cyli. Looks great, please merge!

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_8"></a>|@cyli commented|
|-|-|

Thanks for the review!  Merging.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_9"></a>|@cyli set status to `closed`|
|-|-|

(In [30185]) Merge remove-SafeNetstringReceiver-4546: remove twisted.protocols.basic.SafeNetstringReceiver

Author: cyli
Reviewer: jesstess
Fixes: [#4546](https://github.com/twisted/twisted/issues/4546)

Remove twisted.protocols.basic.SafeNetstringReceiver, which has been deprecated since 2001

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
